controllers/user.js

```diff
@@ -347,9 +347,9 @@ async function getBreachStats(req, res) {
     });
   }
   const fxaResponse = await FXA.verifyOAuthToken(req.token);
-  if (!fxaResponse) {
-    return res.status(404).json({
-      errorMessage: "Cannot find FXA for that OAuth token.",
+  if (fxaResponse.name === "HTTPError") {
```

Comment:
Are we sure `fxaResponse` will be non-null here?
I _assume_ so, looking at the `catch..return` below, but just checking.

Reply:
Yes, there will always be some kind of `fxaResponse` value returned.